driver/normalizer/strconv.go

```diff
@@ -29,15 +54,59 @@ func unquoteSingle(s string) (string, error) {
 	return string(buf), nil
 }
 
+// contains reports whether the string contains the byte c.
+func contains(s string, c byte) bool {
+	for i := 0; i < len(s); i++ {
+		if s[i] == c {
+			return true
+		}
+	}
+	return false
+}
+
+// replaceEscapedMaybe returns a copy of s with "\\old[^0-9]" replaced by new.
+func replaceEscapedMaybe(s string, old, new rune) string {
```

Comment:
Is this logic borrowed from a library function somewhere? If so, can you please mention that in the comment?

(And if not I'm going to look at it much more closely, as it is very fiddly)

Reply:
It is not - this is a custom implementation of looking ahead 2 runes after `\` with additional new tests.